browser/bundles/o-errors.js

```diff
@@ -14,8 +14,8 @@ oErrors.init({
 	tags: {
 		appName: appInfo.name
 	},
-	filterError: function () {
-		return !window.FT.disableOErrors;
+	filterError: function (reportedObject) {
+		return !(window.FT.disableOErrors || reportedObject.error.toString().search('window.FT') > 0);
```

Comment:
maybe that's a little too loose. Would it be better to check for `window.FT.flags || window.FT.nUi`?